src/benchmarks/real-world/Roslyn/CompilerBenchmarks.csproj

```diff
@@ -15,7 +15,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.1.0-beta1-final" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.*" />
```

Comment:
I am not sure about this change. The goal of having the Roslyn benchmarks here was making sure that .NET Core does not regress the performance of Roslyn. With hardcoded Roslyn version it was easy to ensure (every regression could be only .NET Core regression). With variable Roslyn version it could be both: either a Roslyn or .NET Core regression.

Have you discussed this change with @billwert ?

Reply:
Not yet -- I thought that the intention was for our benchmarks to track ongoing perf, not just one build. I was actually thinking we should test against the latest nightly out of our preview branch. Is that not what most other projects are doing? How do you catch regressions otherwise?

Reply:
> I thought that the intention was for our benchmarks to track ongoing perf, not just one build

@agocke Our initial idea was to make sure that the daily builds of .NET Core don't regress the performance. The real-world benchmarks (Roslyn and ML.NET) were supposed to be Contract Tests - something like "this is what we care about, please don't regress our perf".

I agree that it does not solve the problem of detecting regressions in the tested products (Roslyn, ML.NET and more in the future).

Could you please revert this particular change and create a separate issue where we could discuss what would be the best way to approach this problem?

/cc @billwert

Reply:
I agree with @adamsitnik that we should revert this change and come up with a better plan. The intent of having these tests is exactly as stated.

I've opened #896 to track this.

Reply:
pign @billwert